<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Taskbar</class>
 <widget class="QWidget" name="Taskbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>97</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="databaseTab">
      <attribute name="title">
       <string>&amp;Database</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QPushButton" name="devilsButton">
         <property name="text">
          <string>&amp;Devils</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="skillsButton">
         <property name="text">
          <string>&amp;Skills</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mashouButton">
         <property name="text">
          <string>&amp;Mashou</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="itemsButton">
         <property name="text">
          <string>&amp;Items</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>276</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="adminTab">
      <attribute name="title">
       <string>&amp;Administration</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QPushButton" name="usersButton">
         <property name="text">
          <string>&amp;Users</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="importExportButton">
         <property name="text">
          <string>&amp;Import / Export</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>276</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="miscTab">
      <attribute name="title">
       <string>&amp;Miscellaneous</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QPushButton" name="optionsButton">
         <property name="text">
          <string>&amp;Options</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="shoutButton">
         <property name="text">
          <string>&amp;Shoutbox</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logButton">
         <property name="text">
          <string>&amp;Log</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="aboutButton">
         <property name="text">
          <string>&amp;About</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>195</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
